src/Build_Array_from_Permutation/Array_from_Permutation.java

```diff
@@ -0,0 +1,32 @@
+package Build_Array_from_Permutation;
+
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class Array_from_Permutation {
```

Comment:
pls read naming convention for java

[Naming Convention](https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html)